### PR TITLE
fix: adjust hover and selected states

### PR DESCRIPTION
**Pull request by ahmadabdolsaheb**
Checklist:

- [ ] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [ ] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: common/common.scss [hunk 1]
```diff
@@ -23,6 +23,8 @@ $fcc-highlight-color: $tertiary;
 $fcc-highlight-background: fcc-highlight-lightness-switch($fcc-highlight-color);
 $fcc-highlight-mid: mix($fcc-highlight-color, $fcc-highlight-background, 50%);
 $secondary-header-background: mix($header_primary, $header_background, 93%);
+$fcc-d-selected: rgba($primary, 0.16);
+$fcc-d-hover:rgba($primary, 0.08);
```
Comment (ahmadabdolsaheb) [inline common/common.scss:27]:
```suggestion
$fcc-d-hover: rgba($primary, 0.08);
```

File: common/common.scss [hunk 2]
```diff
@@ -807,3 +809,13 @@ div.select-kit-header {
 #list-area .show-more {
   position: relative;
 }
+
+// Adjust hover and read(selected) states
+user-menu .quick-access-panel li {
```
Comment (ahmadabdolsaheb) [inline common/common.scss:814]:
```suggestion
.user-menu .quick-access-panel li {
```

**Comment by ahmadabdolsaheb** [comment]
closes #92

**Comment by ShaunSHamilton** [comment]
Theme has been updated

**Pull request merged**